### PR TITLE
Prepare 2.0 release

### PR DESCRIPTION
**Pull request by Cierpliwy**
* Update README files.
* Update Travis config.
* Update dependencies.

### CODE REVIEW

**Comment by Cierpliwy** [comment]
Yeah, I need to test Expo project, prepare migration file and check latest XCode version.

**Comment by Cierpliwy** [comment]
@dariuszseweryn It's no longer WIP :)

File: README.md [hunk 1]
```diff
@@ -56,73 +59,59 @@ Interested in React Native project involving Bluetooth Low Energy? [We can help
...
-5. Update your `ios/Podfile` to contain:
+4. Update your `ios/Podfile` to contain (it may be already there):
    ```
    pod 'react-native-ble-plx', :path => '../node_modules/react-native-ble-plx'
    ```
-6. Enter `ios` folder and run `pod update`
-7. Minimal supported version of iOS is 8.0
-8. If you want to support background mode:
+5. Enter `ios` folder and run `pod update`
+6. Minimal supported version of iOS is 8.0
```
Comment (dariuszseweryn) [inline README.md:89]: I wonder whether it should be either as a prerequisite even before the first point or at the top of the readme? On the other hand Android integration needs to set `minSdkVersion = 18` which is a valid step. Here it looks like a prerequisite. Maybe make it:
```suggestion
6. Make sure minimal version of iOS is at least 8.0
```

Comment (Cierpliwy) [reply README.md:89]: I'll check if it's even possible to compile RN project with iOS version less then 8.0.

Comment (dariuszseweryn) [reply README.md:89]: [It appears that RN is iOS > 10](https://stackoverflow.com/questions/42200831/which-version-does-react-native-support-ios-and-android)

Comment (Cierpliwy) [reply README.md:89]: Let's remove it then...

Comment (Cierpliwy) [reply README.md]: Updated @dariuszseweryn

File: README.md [hunk 2]
```diff
@@ -86,12 +86,11 @@ Contact us at [Gitter](https://gitter.im/RxBLELibraries/react-native-ble) if you
    pod 'react-native-ble-plx', :path => '../node_modules/react-native-ble-plx'
    ```
 5. Enter `ios` folder and run `pod update`
-6. Minimal supported version of iOS is 8.0
+6. Add `NSBluetoothAlwaysUsageDescription` in `info.plist` file.
```
Comment (dariuszseweryn) [inline README.md:89]:
```suggestion
6. Add `NSBluetoothAlwaysUsageDescription` in `info.plist` file. (it is a requirement since iOS 13)
```

**Pull request merged**